openbas-front/src/admin/components/assets/endpoints/EndpointsDialogAdding.tsx

```diff
@@ -189,13 +189,13 @@ const EndpointsDialogAdding: FunctionComponent<Props> = ({
                             {endpoint.asset_name}
                           </div>
                           <div className={classes.bodyItem} style={{ width: '25%' }}>
-                            <PlatformIcon platform={endpoint.endpoint_platform} width={20} marginRight={10} /> {endpoint.endpoint_platform}
+                            <PlatformIcon platform={endpoint.endpoint_platform} width={20} position={'relative'} marginRight={10} top={5} /> {endpoint.endpoint_platform}
```

Comment:
This component should be atomic.
So, if the component itself is not align (text + icon), it should be changed directly in this component <Platform Icon />

if this is the case, then it is the parent component that must manage to align the interior of this component
Via a display: flex and align-items: center for example.